### PR TITLE
Add info about permission integers

### PR DESCRIPTION
**Pull request by HeyItsGilbert**
## Description

There is nothing specified in the documentation around how to leverage the integer property of ACL permissions.

## Definition of Done

Reference added about how to use permission integers.

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by IanMadd** [comment]
Hey @HeyItsGilbert , can you add your DCO signoff. See here: https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco

File: themes/docs-new/layouts/shortcodes/resources_common_windows_security_acl.md [hunk 1]
```diff
@@ -14,7 +14,10 @@ where
 
 :   Use to specify which rights are granted to the `principal`. The
     possible values are: `:read`, `:write`, `read_execute`, `:modify`,
-    and `:full_control`.
+    `:full_control`, or an integer.
+    
+:    Integers used for permissions must match from the following list
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/resources_common_windows_security_acl.md]:
```suggestion
:    Integers used for permissions must match the following list
```

**Review by kagarmoe — APPROVED** [review]

**Comment by kagarmoe** [comment]
Obvious fix; no DCO needed

**Comment by HeyItsGilbert** [comment]
Thanks for the suggestion and approval @kagarmoe!

**Pull request merged**